### ISSUE TITLE
NO bundled traineddata files

### ISSUE THREAD

**Issue by Shreeshrii** [tesseract4java/tesseract4java#31]
Currently the program includes tesseract binaries but does not include traineddata files. 
Ref - https://github.com/tesseract4java/tesseract4java/issues/19

https://github.com/tesseract4java/tesseract4java/wiki/Usage says

> Before you can start creating new projects, you have to set the environment variable TESSDATA_PREFIX, which points at the directory that contains your "tessdata" directory. Usually this environment variable is set when you install Tesseract.

This requires the user to either install Tesseract or download "tessdata" directory and set the the environment variable TESSDATA_PREFIX.

Installing Tesseract separately can lead to conflict while running program. Asking the user to download "tessdata" is also not straightforward - eg. 

- do  tessconfigs and configs have to be downloaded from https://github.com/tesseract-ocr/tesseract/tree/master/tessdata
- does  osd.traineddata have to be downloaded from https://github.com/tesseract-ocr/tessdata
- eng.traineddata does not work in isolation and requires other eng.* files from https://github.com/tesseract-ocr/tessdata , there is no straightforward way of downloading multiple files from the tessdata repository on windows

-------------------
Hence I would suggest that at a minimum osd.traineddata, eng.* files and other required tessdata files be included with the program and kept in tessdata subdirectory relative to the binaries for the program.

Secondly, I would suggest that program add the option of downloading tessdata files for selected languages (a few other languages have multiple files in tessdata - eg. ara, hin etc) that user can use.

Both VietOCR and Tesseract from UB Mannaheim include these options - please see
- https://sourceforge.net/projects/vietocr/files/vietocr/4.3/
- https://github.com/UB-Mannheim/tesseract/wiki

**Comment by pvorb** [comment]
You're right. This is not straight forward, so I'll try to improve things. Unfortunately I haven't much time left at the moment, so this will take some time.
